app/assets/javascripts/custome.js

```diff
@@ -0,0 +1,3 @@
+$(document).on('change', '.citySelect', function(){
+  $(this).closest('form').trigger('submit');
```

Comment:
@mayur-kambariya Why we have used `.closest` here? Are there multiple forms in the page?